src/restart.js

```diff
@@ -1,4 +1,22 @@
-module.exports = function restart(req, res) {
-  // TODO
-  res.status(501).send('501 Not Implemented');
+let adb = require('./adb');
+let parse = require('url').parse;
+
+module.exports = async function restart(req, res) {
+  let params = parse(req.originalUrl, true /* parseQueryString */).query;
+  let hard;
```

Comment:
Slight risk from someone passing something not readable for `hard`, but they deserve to get an error in that case.